src/feature_alert/label.es

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import {
+  POSITION_TOP,
+  POSITION_BOTTOM,
+  POSITION_LEFT,
+  POSITION_RIGHT,
+} from './constants';
+
+
+const FeatureAlertLabel = ({ position, label, children, ...props }) => {
+  const className = clsx(`c-feature_alert_label is-position-${position}`, props.className);
+  return (
+    <aside {...props} className={className}>
+      {children}
+      <span className="c-feature_alert_label-badge ui-badge ui-badge-rect">{label}</span>
+    </aside>
+  );
+};
+
+FeatureAlertLabel.defaultProps = {
+  position: POSITION_RIGHT,
+};
+
+FeatureAlertLabel.propTypes = {
+  className: PropTypes.string,
+  position: PropTypes.oneOf([
+    POSITION_TOP,
+    POSITION_BOTTOM,
+    POSITION_LEFT,
+    POSITION_RIGHT,
+  ]),
+  label: PropTypes.string.isRequired,
+  children: PropTypes.node,
+};
+
+export default FeatureAlertLabel;
```

Comment:
`Label` ? (file is also called `label`)

Reply:
probably better to leave so, folder is named properly, inner naming would be repetitive and this would leave developer tools still having proper names of the components (more descriptive)